frappe/desk/form/load.py

```diff
@@ -141,7 +141,7 @@ def get_docinfo(doc=None, doctype=None, name=None):
 def add_comments(doc, docinfo):
 	# divide comments into separate lists
 	docinfo.comments = []
-	docinfo.shared = []
+	docinfo.share_logs = []
```

Comment:
Do change this client side instead, the `share_logs` can become `shared` there.

Reply:
> Do change this client side instead, the `share_logs` can become `shared` there.

In file frappe/public/js/frappe/model/perm.js  the variable `shared` is already used in `docinfo` dictionary for getting permission 
```js
_get_perm: (doctype, doc) => {
	...
		if (doc) {
			...
			// apply permissions from shared
			if (docinfo && docinfo.shared) {
				for (const s of docinfo.shared) {
					if (s.user !== user) continue;
					...
	},
```
in the file frappe/desk/form/load.py the variable `shared` was actually being overwritten after calling the `add_comments` method
```py
@frappe.whitelist()
def get_docinfo(doc=None, doctype=None, name=None):
    ...
	add_comments(doc, docinfo)

	docinfo.update(
		{
			...
			"shared": get_docshares(doc),
			...
		}
	)
    ...
```